frontend/pages/policies/ManagePoliciesPage/components/PoliciesTable/PoliciesTableConfig.tsx

```diff
@@ -300,14 +332,37 @@ const generateDataSet = (
   policiesList = policiesList.sort((a, b) =>
     sortUtils.caseInsensitiveAsc(a.name, b.name)
   );
+  // To figure out if the policy has run for all the targeted hosts, we need to do the following calculation:
+  // Each host asynchronously updates its own policy result every `osquery_policy` nanoseconds.
+  // Then, the host count is updated by a cron job on the server every 1 hour (this is hardcoded on the server in `cron.go`).
+  // So, we need to add `osquery_policy` to the time of the cron update.
```

Comment:
This was a bug with `has_run` and tooltip time introduced as part of aggregated policy_stats. The bug was that stats on this page may take up to 2 hours to update (with default configs), and not 1 hour. FYI: @mostlikelee